training/src/anemoi/training/config/small_ens_test.yaml

```diff
@@ -0,0 +1,170 @@
+# defaults:
+# - data: zarr
+# - dataloader: native_grid
+# - diagnostics: evaluation
+# - hardware: example
+# - graph: multi_scale #encoder_decoder_only
+# - model: transformer_ens
+# - training: default
+# - _self_
+
+defaults:
+  - hardware: slurm #example #slurm
+  - data: zarr_aifs-crps
+  - dataloader: ensemble_noeda_aifs-crps
+  - datamodule: single
+  - graph: encoder_decoder_only
+  - model: transformer_ens
+  - training: default
+  - diagnostics: evaluation
+  - _self_
+
+# - graph: encoder_decoder_only
+# - model: transformer
+
+data:
+  resolution: o96
+
+datamodule:
+  _target_: anemoi.training.data.datamodule.AnemoiEnsDatasetsDataModule
+
+hardware:
+  paths:
+    truncation_matrices: /ec/res4/hpcperm/nesl/inter_mat
+    data: /home/mlx/ai-ml/datasets/
+    grids: /home/mlx/ai-ml/grids/
```

Comment:
Grids are no longer needed to pass in the configs so this need to be deleted to also match with the pydantic schemas